### ISSUE TITLE
trimesh.show() invokes non-related code

### ISSUE THREAD

**Issue by halimacc** [mikedh/trimesh#167]
After using trimesh with some of my caffe code, every time I use trimesh.show(), that code will be invoked, even if trimesh is the only imported library and there are only two lines of code:
`mesh = trimesh.load(xxx)`
`mesh.show()`

**Comment by halimacc** [comment]
I relocated the problem while trying to find a minimum reproducible code, it seems mesh.show (or pyglet?) will call `test.py` under the same working directory. 

You could create a file named `test.py` with any code, and runs trimesh.show() under the same working directory.

**Comment by mikedh** [comment]
Hmm, I created a `test.py` that prints on import, and I didn't see its contents run when I did the show call, in either Windows or Linux 

It's interesting though- there appears to be a python package `test`. I don't use it in trimesh, so it seems like it is probably pyglet or someone else trying to import it.

If you replace the `mesh.show()` call with the imports from `viewer.py`, do any of the following reproduce the issue:
```
import pyglet
import pyglet.gl as gl
import numpy as np
import tempfile
import platform
import collections
from trimesh import util
from trimesh.transformations import Arcball
```

Also- what platform/python version are you on?
